tests/unit/frameworks/continuous_learning/multihead/multihead_test.py

```diff
@@ -28,18 +28,38 @@
 import unittest
 
 from torch.utils.data import DataLoader
+from torchvision import transforms
 
-from nupic.research.frameworks.continuous_learning.multihead.subdataset import (
-    get_datasets,
+from nupic.research.frameworks.continuous_learning.multihead.multihead import (
+    get_active_classes,
 )
+from nupic.research.frameworks.pytorch.dataset_utils.dataset_utils import split_dataset
+from nupic.research.frameworks.pytorch.test_utils.fake_data import (
+    FakeDataPredefinedTargets,
+)
+
+
+def setup(scenario):
+    fake_train_dataset = FakeDataPredefinedTargets(image_size=(1, 28, 28))
+    fake_test_dataset = FakeDataPredefinedTargets(image_size=(1, 28, 28))
```

Comment:
Why not just `FakeData`?

Reply:
(same comment as above)

because the `split_dataset` function assumes the dataset class has a `targets` attribute (which MNIST does), but the `FakeData` class does not.

Reply:
Maybe the fix should be in the `split_dataset` function instead

Reply:
reverted changes to `split_dataset`, so it no longer uses `target_transform`.